### ISSUE TITLE
Support compressed SAS file

### ISSUE THREAD

**Issue by leopoldsw** [Roche/pyreadstat#66]
Is there a plan to add support for compressed sas file? Thanks.

**Comment by ofajardo** [comment]
The feature has been requested in readstat (https://github.com/WizardMac/ReadStat/issues/21) but stale since 5 years. We need the fix in readstat.

**Comment by evanmiller** [comment]
@ofajardo Please try https://github.com/WizardMac/ReadStat/tree/sas-binary-compression

**Comment by ofajardo** [comment]
@leopoldsw Can you please provide a sample file?

**Comment by ofajardo** [comment]
tested OK in pyreadstat with the attached sample file I generated myself like this:

```
data SAMPLES.sample_bincompressed(compress=binary);
set SAMPLES.sample;
run;
```

The file is stored permanently in the pyreadstat repo in the test_data/basic/sample_bincompressed.sas7bdat, for now in the sasbin_dev branch.

[sample_bincompressed.sas7bdat.zip](https://github.com/Roche/pyreadstat/files/5128514/sample_bincompressed.sas7bdat.zip)

@leopoldsw if you like you can still provide a file of yourself for me to test or you can test yourself by cloning and compiling the branch sasbin_dev

**Comment by ofajardo** [comment]
fixed in version 1.0.2
